_posts/2024-04-14-blog.md

```diff
@@ -0,0 +1,86 @@
+I picked a set of candidate present-day individuals (plus the 'Loschbour' hunter-gatherer) whose positions roughly spanned the area of each PCA plot.
+I then randomly added missing data to each candidate individual's genotypes at a given rate, creating 200 replicates per missingness rate, per individual.
+
+I focused on four PCA setups that are somewhat prevalent in human aDNA literature:
+
+-   Worldwide PCA
+
+-   Eurasian PCA (à la [Jeong2019](http://dx.doi.org/10.1534/genetics.112.145037))
+
+-   West Eurasian PCA (à la [Lazaridis et al. 2014](http://dx.doi.org/10.1038/nature13673))
+
+-   African PCA (à la [Skoglund et. al 2017](http://dx.doi.org/10.1016/j.cell.2017.08.049))
+
+## Results
+
+<sup>(You can right click on any of the images below and select "Open Image in a New Tab" to see them in more detail.)</sup>
+
+### Missing 90% of all genotypes (60.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.9.png)| ![Eurasian](/assets/media/event_images/2024-04-14-blog/Eurasia.m0.9.png)
+![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.9.png)|![African](/assets/media/event_images/2024-04-14-blog/Africa.m0.9.png)
+
+### Missing 97% of all genotypes (18.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.97.png) |![Eurasian](/assets/media/event_images/2024-04-14-blog/Eurasia.m0.97.png)
+![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.97.png) |![African](/assets/media/event_images/2024-04-14-blog/Africa.m0.97.png)
+
+### Missing 99% of all genotypes (6.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.99.png) |![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.99.png)
+
+## Conclusions
+
+In literature, the chosen cutoffs are both sensible and (strictly speaking) arbitrary, commonly ranging between 10.000 and 20.000 covered SNPs.
+Any individuals with fewer than that number of SNPs are often excluded from downstream analyses.
+Looking at the results above, you can see why:
+
+ - High degrees of missingness can affect the projected placement of an individual in regional PCAs. 
+    - e.g. with sparse enough data (99% missing), some replicates of the Sicilian individual 'TP08' are projected closer to present-day Greeks than the other Sicilians in the dataset.
+ - While the exact projection of low-coverage individuals in the Worldwide PCA is also affected, the spread of replicates shows that it is still perfectly possible to access if a studied individual is genetically more closely related to present-day African, South Asian or Native American populations from the PCA results, even from only a few thousand SNPs.
+ - Comparing the spread of replicates around 'Loschbour' to those around the other candidate individuals, it does not look like projection of the candidate individual itself affects the spread of the replicates. (n=1!!)
```

Comment:
Clarify/rewrite this

Reply:
Agree, I would put point 1 in it's own 'list'.

You sort of have two categories of outcomes:

1. For in-depth technical analyses, there is quite a lot of variatoin in placement (which is why there are cut offs and these are important)

But at the same time

1. The variation isn't enough to give you a _rough_ idea of where it falls in broader conexts
2. The candidate project doesn't effect spread of replicates (I don't get what you mean by this one, but I'm not a poop-genner)